app/models/recommendation.py

```diff
@@ -12,10 +12,11 @@
 # Needs to exist for pydantic to resolve the model field "item: ItemModel" in the RecommendationModel
 from app.graphql.item import Item
 from app.models.candidate_set import candidate_set_factory
+from app.models.metrics.firefox_new_tab_metrics_factory import FirefoxNewTabMetricsFactory
 from app.models.metrics.recommendation_metrics_factory import RecommendationMetricsFactory
 from app.models.item import ItemModel
 from app.models.slate_experiment import SlateExperimentModel
-from app.rankers import get_ranker, THOMPSON_SAMPLING_RANKERS
+from app.rankers import get_ranker, THOMPSON_SAMPLING_RANKERS, FIREFOX_THOMPSON_SAMPLING_RANKERS
```

Comment:
Naming stuff like `A_THING` and `MORE_QUALIFIERS_A_THING` to me suggests that the second one is a subset of the first one. If they are peers (as we have here with `THOMPSON_SAMPLING_RANKERS` and `FIREFOX_THOMPSON_SAMPLING_RANKERS`), we can indicate that by putting qualifiers on both of their names.